### DISCUSSION TITLE
Self hosted chatwoot with Ai chatbot

### DISCUSSION THREAD

**Discussion by nathanieltong-1127** [chatwoot/chatwoot#9601] [Q&A]
Currently trying to set up an Ai chatbot to respond to customers, using this tutorial (https://latenode.com/templates/how-to-make-no-code-ai-chatbot-for-your-website) [if there are other suggestions to set this up differently works as well] but I still cant get any responses from the OpenAi assistant, not sure where the issue is.

**Comment by dbim85** [comment]
Jump on Latenode's Discord channel - and they will help you to handle this)
https://go.latenode.com/fromwebsitetodiscord
